indexer/services/comlink/src/controllers/api/v4/affiliates-controller.ts

```diff
@@ -9,25 +9,95 @@ import { getReqRateLimiter } from '../../../caches/rate-limiters';
 import config from '../../../config';
 import { handleControllerError } from '../../../lib/helpers';
 import { rateLimiterMiddleware } from '../../../lib/rate-limit';
+import { handleValidationErrors } from '../../../request-helpers/error-handler';
 import ExportResponseCodeStats from '../../../request-helpers/export-response-code-stats';
-import { AffiliateReferralCodeRequest, AffiliateReferralCodeResponse } from '../../../types';
+import {
+  AffiliateAddressRequest,
+  AffiliateReferralCodeRequest,
+  AffiliateReferralCodeResponse,
+  AffiliateAddressResponse,
+  AffiliateSnapshotResponse,
+  AffiliateSnapshotResponseObject,
+  AffiliateSnapshotRequest,
+  AffiliateTotalVolumeResponse,
+  AffiliateTotalVolumeRequest,
+} from '../../../types';
 
 const router: express.Router = express.Router();
 const controllerName: string = 'affiliates-controller';
 
+// TODO(OTE-731): replace api stubs with real logic
 @Route('affiliates')
 class AffiliatesController extends Controller {
   @Get('/referral_code')
   async getReferralCode(
     @Query() address: string, // eslint-disable-line @typescript-eslint/no-unused-vars
   ): Promise<AffiliateReferralCodeResponse> {
-    // TODO: OTE-731 replace apit stubs with real logic
     // simulate a delay
     await new Promise((resolve) => setTimeout(resolve, 100));
     return {
       referralCode: 'TempCode123',
     };
   }
+
+  @Get('/address')
+  async getAddress(
+    @Query() referralCode: string, // eslint-disable-line @typescript-eslint/no-unused-vars
+  ): Promise<AffiliateAddressResponse> {
+    // simulate a delay
+    await new Promise((resolve) => setTimeout(resolve, 100));
+    return {
+      address: 'some_address',
+    };
+  }
+
+  @Get('/snapshot')
+  async getSnapshot(
+    @Query() offset ?: number,
+      @Query() limit ?: number,
+      @Query() sortByReferredFees ?: boolean,
```

Comment:
```suggestion
    @Query() offset?: number,
      @Query() limit?: number,
      @Query() sortByReferredFees?: boolean,
```
super small nit, no need for spaces before the `?` for parameters

Reply:
oops, good catch. I assumed linter would catch this stuff but guess not.